### ISSUE TITLE
Recognize all contributors to the InnerSource Patterns

### ISSUE THREAD

**Issue by spier** [InnerSourceCommons/InnerSourcePatterns#219]
It would be nice to recognize all contributors to our InnerSource Patterns in a public way.

A couple of ideas why that might be useful:
- people that have committed code to this repo are already visible [here](https://github.com/InnerSourceCommons/InnerSourcePatterns/graphs/contributors) however that is a bit hidden
- there might be contributors that have helped in ways other than sending pull requests

### Some tools I found:

- There is the [allcontributors.org](https://allcontributors.org/) project that offers a way to recognize all contributors and list them in the README. They also have a bot 👍 
- a github action with a slightly simpler idea to accomplish this https://github.com/marketplace/actions/contribute-list

### Side note:

Listing all contributors in the book would be also nice (once we publish that :))

**Comment by lenucksi** [comment]
Good idea. I've already seen the allcontributors.org bot elsewhere and it looks nice.

Also, having all participants included in the author notes of publications is good practice we've always tried to follow in the learning path WG. What we did though was to check with people if they want to be included in the check after creating the contributor list.
